app/stac_api/templates/js/admin/collection_help_search.js

```diff
@@ -0,0 +1,58 @@
+window.onload = function() {
```

Comment:
Code style (indentation issues, line length, ...)

Reply:
should be better now.